src/System.Private.ServiceModel/src/System/ServiceModel/EndpointIdentity.cs

```diff
@@ -148,7 +149,7 @@ internal static EndpointIdentity ReadIdentity(XmlDictionaryReader reader)
                 {
                     throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new XmlException(SR.Format(SR.UnrecognizedIdentityType, reader.Name, reader.NamespaceURI)));
                 }
-                throw ExceptionHelper.PlatformNotSupported("EndpointIdentity.ReadIdentity RsaEndpointIdentity is not supported.");
+                reader.ReadEndElement();
```

Comment:
Question: Does this ReadEndElement plus the one at the end of the method cause a problem?  Or are the 2 ReadEndElements necessary?